File: MODUL 3 PEMROGRAMAN 1/PRAK305-2310817120002-SITIRATNADWINTASARI.c
#include <stdio.h>
int main(){
    int d,hr,jm,mn,dt;
    for (int i = 0; i < 5; i++){
    scanf("%d",&d);
    hr = d / 84600;
    jm = d / 3600;
    mn = (d - jm * 3600)/60;
    dt = (d - jm * 3600) - (mn * 60);
    if (hr >=1){
        printf("%d hari %.2d:%.2d:%.2d\n", hr, jm %24,mn, dt);
    } else {
        printf("%.2d:%.2d:%.2d\n",jm, mn, dt);
    }
    } return 0;
}

File: MODUL 3 PEMROGRAMAN 1/PRAK305-2310817120002-SITI RATNA DWINTA SARI.py
for i in range(5):
    d = int(input())
    hr = d // 86400
    jm = (d // 3600) % 24
    mn = (d // 60) % 60
    dt = d % 60
    if hr >= 1:
        print(f"{hr} hari {jm:02d}:{mn:02d}:{dt:02d}")
    else:
        print(f"{jm:02d}:{mn:02d}:{dt:02d}")
